app/src/androidTest/java/org/mozilla/focus/helpers/TestHelper.java

```diff
@@ -222,7 +209,16 @@ public static String getMenuItemText(UiObject item) throws UiObjectNotFoundExcep
             .resourceId(getAppName() + ":id/refresh")
             .enabled(true));
 
-    private TestHelper () throws UiObjectNotFoundException {
+    public static void expandNotification() throws UiObjectNotFoundException {
+        if (!notificationOpenItem.waitForExists(waitingTime)) {
+            if (!notificationExpandSwitch.exists()) {
+                notificationBarDeleteItem.pinchOut(50,5);
+            }
```

Comment:
\[Checkstyle\] ERROR: '\}' at column 13 should be on the same line as the next part of a multi\-block statement \(one that directly contains multiple blocks: if/else\-if/else, do/while or try/catch/finally\)\.